OpenEdXMobile/src/main/java/org/edx/mobile/module/analytics/Analytics.java

```diff
@@ -420,4 +420,27 @@ void courseDetailShared(@NonNull String courseId, @NonNull String aboutUrl,
         String PROFILE_VIEWED = "Viewed a profile";
         String PROFILE_PHOTO_SET = "Set a profile picture";
     }
+
+    /**
+     * We can't have concrete functions inside interfaces till Java 8, therefore this
+     * class has been defined to add static utilities to this interface.
+     */
+    class Util {
+        /**
+         * Resolves and gives back the string alternative of the given share type.
```

Comment:
"gives back" can be changed to "returns". Maybe the description can be supplanted by just the `@return` annotation: "`@return` **The resolved** string alternative of the given share type."